### PR TITLE
gift rains

### PR DESCRIPTION
**Pull request by blaumeise20**

### CODE REVIEW

**Comment by milenakos** [comment]
what

**Comment by milenakos** [comment]
also get_or_create returns the object as well as bool of whether it was getted or created so do `user, _ = User.get_or_create()`

**Comment by blaumeise20** [comment]
right ok i forgor will fix it

**Comment by blaumeise20** [comment]
done

**Comment by milenakos** [comment]
i will look into this tomorrow its 1:20 am

File: main.py [hunk 1]
```diff
@@ -2293,89 +2296,168 @@ async def ping(message: discord.Interaction):
...
-            myview.add_item(button)
-            myview.add_item(button2)
+                async def pay(interaction):
+                    if interaction.user.id == message.user.id:
+                        await interaction.response.defer()
+                        try:
+                            await interaction.edit_original_response(view=None)
+                        except Exception:
+                            pass
+                        actual_user.rain_minutes -= tax_amount
```
Comment (milenakos) [inline main.py:2417]: get the user again before doing that to prevent weird shit

Comment (milenakos) [reply main.py:2417]: okay now i realize its not done for cat taxes either so maybe its fine?
i think its safer to also add it there then

Comment (blaumeise20) [reply main.py:2417]: ok i'll look at it later, don't think there is an issue caused by that though

**Comment by blaumeise20** [comment]
ok cool

**Pull request merged**